_posts/2015-02-16-Utilising-API-Blueprint-in-API-Clients.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Utilising API Blueprint in API Clients
+excerpt: Imagine an API client that can use the already-defined information from an API Blueprint to build a request
+layout: post
+date: 2015-02-16 18:00:00 +0000
+author: kyle
+published: true
+comments: true
+---
+
+I’m excited to release a new way to interface to API Blueprints. A client-side framework in Swift that allows you to introspect and perform actions at run-time.
+
+Imagine an API client that can use the already-defined information from an API Blueprint to build a request rather than hardcoding that information. This allows you to start building smarter clients which can evolve at run-time based on the blueprint.
+
+The SDK is created using a framework we’ve built which implements the [Representor Pattern](https://github.com/the-hypermedia-project/charter), a canonical representation of hypermedia messages across multiple media types and languages. One of the key constraints to Representational State Transfer (REST) is Hypermedia as the Engine of Application State (HATEOAS). This constraint requires the message from the server to contain information about how to transition to other states along with links to other resources, specifically through hypermedia.
```

Comment:
Should this be: https://github.com/the-hypermedia-project/charter#representor-pattern